internal/service/domain/tree/import_tree_test.go

```diff
@@ -0,0 +1,193 @@
+package tree
+
+import (
+	"context"
+	"github.com/green-ecolution/green-ecolution-backend/internal/entities"
+	service "github.com/green-ecolution/green-ecolution-backend/internal/service/_mock"
+	storageMock "github.com/green-ecolution/green-ecolution-backend/internal/storage/_mock"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"testing"
+	"time"
+)
+
+func TestTreeService_ImportTree(t *testing.T) {
+	ctx := context.Background()
+
+	treeToImport := &entities.TreeImport{
+		Latitude:     54.801539,
+		Longitude:    9.446741,
+		PlantingYear: 2023,
+		Species:      "Oak",
+		Number:       "T001",
+	}
+	t.Run("should create a new tree when no matching tree exists", func(t *testing.T) {
```

Comment:
Maybe we can add a test case if the creating fails?

Reply:
I think we don't need to test the failure cases for creation because we have already covered them in `tree_test.go`.